js_tests/gis/mapwidget.test.js

```diff
@@ -0,0 +1,40 @@
+/* global module, test */
+/* eslint global-strict: 0, strict: 0 */
+'use strict';
+
+module('gis.OLMapWidget');
+
+test('MapWidget.featureAdded', function(assert) {
+    var options = {id: 'id_point', map_id: 'id_point_map', geom_name: 'Point'};
+    var widget = new MapWidget(options);
+    assert.equal(widget.layers.vector.features.length, 1);
+    assert.equal(widget.layers.vector.features[0].geometry.toString(), 'POINT(7.8177 47.397)',
```

Comment:
If it were me, I'd use a similar hanging indent style as we do in Python instead of a non-multiple of 4 indent:
```
assert.equal(
    widget.layers.vector.features[0].geometry.toString(),
    'POINT(7.8177 47.397)',
    'Point addded to vector layer'
);
```